akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -322,6 +329,11 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
   val -> = FSM.->
 
   /**
+   * Unicode alias for ->, matches a (S, S) pair
+   */
+  val → = FSM.->
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name '?' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/vasseurjb/akka/file/2046658285/issues/source?bid=3132306&fileBranchId=3132374#l334)